apps/cowswap-frontend/src/modules/swap/pure/warnings.tsx

```diff
@@ -98,13 +100,15 @@ export const SwapWarningsTop = React.memo(function (props: SwapWarningsTopProps)
         <BundleTxSafeWcBanner nativeCurrencySymbol={nativeCurrencySymbol} supportsWrapping />
       )}
 
-      <TwapSuggestionBanner
-        chainId={chainId}
-        priceImpact={priceImpact}
-        buyingFiatAmount={buyingFiatAmount}
-        tradeUrlParams={tradeUrlParams}
-        sellAmount={trade?.inputAmount.toExact()}
-      />
+      {showTwapSuggestionBanner && (
+        <TwapSuggestionBanner
+          chainId={chainId}
+          priceImpact={priceImpact}
+          buyingFiatAmount={buyingFiatAmount}
+          tradeUrlParams={tradeUrlParams}
+          sellAmount={trade?.inputAmount.toExact()}
+        />
+      )}
```

Comment:
My suggestion was more, if `enabledTradeTypes` had `TradeType.ADVANCES` ifhttps://github.com/cowprotocol/cowswap/blob/6e6352f718d7ce18a0e5ce4fc88dc29c418e33ce/libs/widget-lib/src/types.ts#L220

Do you think is better to not suggest it for all. That's just a way to advertise the feature even for EOAs

Reply:
I got your point, fixed